examples/typescript/invoke-function.ts

```diff
@@ -835,8 +835,13 @@ async function invokeFunctionHelper(
   client.endpoint = fnSummary.invokeEndpoint!;
   const functionId = fnSummary.id || "";
 
+  if(payload && (typeof payload === 'object')) {
```

Comment:
Will `payload` ever be of type `object` in this case? The type declaration for the `payload` parameter is `string`. An alternative would be to change the type declaration for it to `string | object` in this function (which will have to propagate upwards to `invokeFunction`). You would also have to change line 844 to `invokeFunctionBody: payload as string` to enforce it.

Aside from that, the typescript changes are good (tested it).